assets/scripts/segments/segment-dict.js

```diff
@@ -0,0 +1,233 @@
+import SEGMENT_INFO from './segment-info.json'
```

Comment:
Documentation in the PR conversation tab is definitely helping understand how things are structured. I'm thinking of renaming `segment-info.json` to `segment-components.json` or maybe `components.json` for a shorter file name, and similarly `SEGMENT_INFO` becomes `SEGMENT_COMPONENTS`. Thoughts?

Reply:
Sure, sounds good - more understandable. 
What about `segment-dict.js`? I'm not quite sure what to name it but eventually it'll replace `info.js` I think?

Reply:
Yeah, until then I think we can leave the filename as it is.